flags.go

```diff
@@ -219,12 +219,6 @@ func newApp() (app *cli.App) {
 					"inodes.",
 			},
 
-			cli.DurationFlag{
```

Comment:
aren't these changes already done.

Reply:
I did it from the same branch that's why whole changes came.